dockerfiles/run/guice/provisioned/startup.sh

```diff
@@ -2,4 +2,4 @@
 
 wait-for-it.sh --host=localhost --port=9999 --strict --timeout=0 -- ./initialdata.sh &
 
-java -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ -jar james-server.jar
\ No newline at end of file
+java -classpath '/root/james-server.jar:/root/james-server-jpa-guice.lib/*' -javaagent:/root/james-server-cli.lib/openjpa-2.2.1.jar -Dlogback.configurationFile=/root/conf/logback.xml -Dworking.directory=/root/ org.apache.james.JPAJamesServerMain
```

Comment:
It's a copy and paste from jpa image.

Sorry but I won't fix here

(It's because @quynhn  initially added the driver as a build dependency, then @mbaechler  complained. She moved away from it, but relied on this classpath trick)

To fix it, we should also fix the jpa - guice image. Not that easy. And in the meantime, the demo image is broken. I think fixing this issue should be done in a second time.